app/views/posts/index.html.erb

```diff
@@ -22,24 +35,24 @@
     <% else %>
       <% @posts.each do |post| %>
         <div class="col-sm-12 border rounded my-1">
-          <h5 class="user-name py-3">
-            <%= link_to "Post #{post.id}", user_post_path(@user, post), title: "To Post Details" %>
-          </h5>
-          <h3 class="user-name">
+          <h6 class="user-name pt-3">
+            <%= link_to "Post #{post.id}", user_post_path(@user, post), title: "To Post Details", class: "text-muted" %>
+          </h6>
+          <h3>
             <%= post.title %>
           </h3>
-          <p class="user-name">
+          <p>
             <%= post.text %>
           </p>
         </div>
 
         <div class="col-sm-12 border rounded mb-5">
-          <h4 class="user-name">Comments:</h4>
+          <h4>Comments:</h4>
           <% if post.comments.empty? %>
-            <p class="user-name">This post has no comments.</p>
+            <p>This post has no comments.</p>
           <% else %>
             <% post.comments.each do |comment| %>
-              <p class="user-name">
+              <p>
```

Comment:
Your design
![image](https://github.com/sergio-usma/rails_blog_pj/assets/78251927/842c832b-0a44-4f74-be75-e5984869a54e)

- [ ] According to the wireframe, kindly use `post.recent_comments` to render 5 most recent comments in the `app/views/posts/index.html.erb` page 🤗🛠

Wireframe
![image](https://github.com/sergio-usma/rails_blog_pj/assets/78251927/b881d95a-19f0-4dc3-83c2-a4fafac07dea)